standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -5144,6 +5159,7 @@ private int abortTxns(Connection dbConn, List<Long> txnids, boolean checkHeartbe
       if (MetastoreConf.getBoolVar(conf, MetastoreConf.ConfVars.METASTORE_ACIDMETRICS_EXT_ON)) {
         Metrics.getOrCreateCounter(MetricsConstants.TOTAL_NUM_ABORTED_TXNS).inc(txnids.size());
       }
+      LOG.info("Txns are aborted successfully due to : {}", errorMsg);
```

Comment:
Could we populate TXN_META_INFO with aborted reason as well?
see //todo: should make abortTxns() write something into TXNS.TXN_META_INFO about this

Reply:
Simhadri wanted to add a new column in TXNS table for that, but looks like we already have one TXN_META_INFO. In some situations that might be useful, but I'll leave it up to you.

Reply:
I have added the message contents in this column. Done.